packages/frontend/src/components/RegisterForms/ProjectConnectFlow.styles.tsx

```diff
@@ -18,12 +18,7 @@ const bounceIn = keyframes`
     }
 `;
 
-export const StyledSuccessIcon = styled(Icon)`
-    margin: 40px 0;
+export const SuccessIconBounce = styled(MantineIcon)`
     animation-name: ${bounceIn};
     animation-duration: 0.7s;
```

Comment:
kept this because: 
couldn't work with `Transition` from mantine: https://v6.mantine.dev/core/transition/